src/SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Diagnostics;
+
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob.DependencyResolution;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.ContractAgreements.WebJob
+{
+    // To learn more about Microsoft Azure WebJobs SDK, please see http://go.microsoft.com/fwlink/?LinkID=320976
+    class Program
+    {
+        // Please set the following connection strings in app.config for this WebJob to run:
+        // AzureWebJobsDashboard and AzureWebJobsStorage
+        static void Main()
+        {
+            // The following code ensures that the WebJob will be running continuously
+            //var host = new JobHost();
+            //host.RunAndBlock();
+
+            try
+            {
+                var container = IoC.Initialize();
+                var logger = container.GetInstance<ILog>();
+                logger.Info("ContractAgreements job started");
+                var timer = Stopwatch.StartNew();
+
+                var service = container.GetInstance<ProviderAgreementStatusService>();
+                service.UpdateProviderAgreementStatuses().Wait();
+
+                timer.Stop();
+
+                logger.Info($"ContractAgreements job done, Took: {timer.ElapsedMilliseconds} milliseconds");
+            }
+            catch (Exception ex)
+            {
```

Comment:
should we be using the azure web job error handling stuff? some links on here: http://stackoverflow.com/questions/35928286/how-to-intercept-an-azure-webjob-failure-exception